hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/app.less

```diff
@@ -34,7 +35,7 @@
 }
 
 .page-header {
-  padding: 10px 0;
+  padding: 20px 0;
```

Comment:
Any reason of changing this? If it's not related to the dependency migration let's address it in follow up ticket.

Reply:
Sure, thanks Ivan. It was just changed to add a bit more spacing in the heading element.
But even the previous change works. Reverting in the latest commit